src/parse.ts

```diff
@@ -189,10 +204,25 @@ const parseRelationship = (variableName: string, originalObject: string): Formul
     })
 }
 
+const parseInstance = (variableName: string): FormulaIdentifierInfo[] => {
+  if ((variableName.match(/\./g) || []).length > 2) {
+    return []
+  }
+  if (getObject(variableName).endsWith(RELATIONSHIP_SUFFIX)) {
+    return []
+  }
+  return [{
+    type: 'instance',
+    instance: isGlobalInstance(variableName) ? getField(variableName) : variableName,
+  }]
+}
+
 export const parseFormulaIdentifier = (variableName: string, originalObject: string): FormulaIdentifierInfo[] => {
   // this order matters, we have to evaluate object types before anything else because the syntax can be extremely
   // similar to other types
-
+  if (isGlobalInstance(variableName)) {
+    return parseInstance(getField(variableName))
+  }
```

Comment:
What about the comment above? Also isn't most of the logic above encapsulated in the parseInstance function

Reply:
I believe we still produce the right results, because the `Global` and `$ObjectType` prefixes are distinct.
The reason we want an explicit check for `Global` is that if we see this prefix then we know it has to be an instance, there's no need for the instance/field ambiguity.